crates/astria-sequencer/src/transaction.rs

```diff
@@ -78,24 +64,3 @@ impl ActionHandler for Transaction {
         }
     }
 }
-
-#[cfg(test)]
-mod test {
-    use hex;
-
-    use super::*;
-
-    #[test]
-    fn test_transaction() {
-        let tx = Transaction::new_accounts_transaction(
-            Address::from("bob"),
-            Address::from("alice"),
-            Balance::from(333333),
-            Nonce::from(1),
-        );
-        let bytes = tx.to_bytes().unwrap();
-        let tx2 = Transaction::from_bytes(&bytes).unwrap();
-        assert_eq!(tx, tx2);
-        println!("0x{}", hex::encode(bytes));
-    }
-}
```

Comment:
is this moved somewhere else?

Reply:
This was basically testing if the serde deser worked so it's redundant in that sense.